### ISSUE TITLE
Style/TrivialAccessors false positive

### ISSUE THREAD

**Issue by mockdeep** [bbatsov/rubocop#1653]
We're getting a violation from `Style/TrivialAccessors` when the method name doesn't match the object it's returning:

```ruby
def foo
  @blah
end
```

```
C: Style/TrivialAccessors: Use attr_reader to define trivial reader methods
```

**Comment by mockdeep** [comment]
nm, just saw [this](https://github.com/bbatsov/rubocop/issues/1605).
